app/src/main/java/com/iron/espresso/presentation/home/study/list/StudyListViewModel.kt

```diff
@@ -30,8 +81,14 @@ class StudyListViewModel @ViewModelInject constructor(private val studyApi: Stud
             .networkSchedulers()
             .subscribe({
                 Logger.d("$it")
-                _studyList.value = it.data?.map { studyResponse ->
-                    studyResponse.toStudyItem()
+                if (it.data != null) {
```

Comment:
```suggestion
                if (!it.data.isNullOrEmpty()) {
```
empty 체크 추가가 필요할듯. 0개내려오면 기존 목록이 사라지게됨

Reply:
수정하였습니다!